tests/integration/dbclient/appl/pom.xml

```diff
@@ -0,0 +1,417 @@
+                </plugin>
+            </plugins>
+        </pluginManagement>
+
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-libs</id>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.jboss.jandex</groupId>
+                <artifactId>jandex-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>make-index</id>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <parallel>methods</parallel>
+                    <threadCount>10</threadCount>
+                    <systemPropertyVariables>
+                        <junit.jupiter.extensions.autodetection.enabled>true</junit.jupiter.extensions.autodetection.enabled>
+                    </systemPropertyVariables>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>test</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
+                        </goals>
+                        <configuration>
+                            <includes>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.simple.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.transaction.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.statement.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.metrics.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.result.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.mapping.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.interceptor.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.health.*IT</include>
+                            </includes>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+
+    </build>
+
+    <profiles>
+
+        <profile>
+            <id>docker</id>
```

Comment:
Please remove this profile.

Reply:
I'd like to keep it. It's helping me a lot when working on the tests.

Reply:
Removed.